inspirehep/modules/search/api.py

```diff
@@ -77,14 +78,20 @@ def mget(self, uuids, **kwargs):
         :type uuids: list of strings representing uuids
         :returns: list of JSON documents
         """
-        documents = current_search_client.mget(
-            index=self.Meta.index,
-            doc_type=self.Meta.doc_types,
-            body={'ids': uuids},
-            **kwargs
-        )
-
-        return [document['_source'] for document in documents['docs']]
+        results = []
+
+        try:
+            documents = current_search_client.mget(
+                index=self.Meta.index,
+                doc_type=self.Meta.doc_types,
+                body={'ids': uuids},
+                **kwargs
+            )
+            results = [document['_source'] for document in documents['docs']]
+        except RequestError as e:
+            current_app.logger.warning('{}'.format(e))
```

Comment:
This is not the right way to do logs. See: https://github.com/inspirehep/inspire-next/issues/1371